app/src/main/java/com/intervaltimer/google/services/TimerService.java

```diff
@@ -75,7 +75,7 @@
     //Timer Flags
     private boolean isBlockPeriodization = false;
     private boolean isStarttimer = false;
-    private boolean isWorkout = false;
+    private boolean isWorkout = false; // If there is workout in progress (as opposed to rest)
```

Comment:
I'm not sure if I understand the consequences of this comment. What is "opposed to rest"?

Reply:
There are two possible understanding of the term "workout", which I find confusing. One is that there is an active workout in process in any stage, one is that it is specifically in the "workout" interval and not the "rest" interval. Here, it is the latter.